popSimManu.tex

```diff
@@ -458,6 +460,25 @@ \section*{Discussion}
 we have released as part of \stdpopsim should provide a cookbook for the community to
 dive in to more rigorous benchmarking than we have done thus far.
 
+Although we did not produce an exhaustive comparison of different demographic inference methods,
+our results are informative,
+and highlight the utility of easy comparison of methods on the same footing.
+The tests here were under best-case scenarios --
+simulations of discrete, randomly mating populations that fit the assumptions of the methods nearly exactly --
+and yet different methods, as well as applications of the same method
+to different simulations of the same model,
+produced somewhat different results.
```

Comment:
Delete "somewhat"?

Reply:
Also, should we mention what happens when we run simulations for the right models but the wrong species? Being able to easily try out methods across different species is a major bonus I think.

Reply:
Let's see - I said "somewhat" because I didn't want to be overly dramatic about it.  

And, the different species is a good point, but may be getting into the weeds a bit - to many people it would sound wierd to simulate humans with a Drosophila demographic history.

So, I'm not inclined to change these things?

Reply:
OK, sounds good. "Somewhat" reads a bit oddly to me, but I can't think of a better work.